lib/features/drawing_detail/drawing_detail_view.dart

```diff
@@ -46,8 +48,23 @@ class DrawingDetailScreenState extends State<DrawingDetailScreen> {
   @override
   Widget build(BuildContext context) {
     final accountContextState = context.watch<AccountContextBloc>().state;
+    final drawingDetailState = context.watch<DrawingDetailBloc>().state;
     final drawingsCatalogBloc = context.read<DrawingsCatalogBloc>();
 
+    if (accountContextState is AccountContextLoadedState && drawingDetailState is DrawingDetailStateLoaded) {

```

Comment:
It is okay to add the logging here. 
But there is a better place to add it. 
User could open the sheet from the catalog or from the navigation rail, which contains the list of drawings. 
A better place would be the case of `loadSheet` which can capture all the cases.